decidim-comments/app/packs/src/decidim/comments/comments.js

```diff
@@ -1 +1,13 @@
-import "src/decidim/comments/comments.component"
+import CommentsComponent from "src/decidim/comments/comments.component"
+
+window.Decidim.CommentsComponent = CommentsComponent;
+
+$(() => {
```

Comment:
This block causes the flaky test that you added the `sleep 5` to. Even with the sleep this is currently flaky at the CI.

The problem with this is that now in the legacy JS, the comments component is mounted twice. Once in this code and once here:
https://github.com/decidim/decidim/blob/425b6b2726931e5b51a7337c776c4cee55cb1e8f/decidim-core/app/packs/src/decidim/index.js#L91-L96

You can try the flaky test as follows:
```bash
$ cd decidim-accountability
$ for i in {1..20}; do bundle exec rspec spec/system/comments_spec.rb -e "when authenticated when a user deletes a comment when the comment is authored by user the user can delete the comment and updates the comments counter" ; done
```

If it passes 100% of the times, it should be fixed.

Reply:
Oh, thank you very much, I was going crazy with this test